modules/install-sno-monitoring-the-installation-manually.adoc

```diff
@@ -8,11 +8,15 @@
 
 Use `openshift-install` to monitor the progress of the single-node cluster installation.
 
+.Prerequisites
+
+* Ensure that the boot drive order in the server BIOS settings defaults to booting the server from the target installation disk.
+
 .Procedure
 
 . Attach the modified {op-system} installation ISO to the target host.
 
-. Configure the boot drive order in the server BIOS settings to boot from the attached discovery ISO and then reboot the server.
+. Configure the server to boot from the {op-system-first} discovery ISO image. The discovery ISO image writes the {op-system-first} system configuration to the target installation disk and automatically triggers a server restart.
```

Comment:
Same comment about the discovery ISO running RHCOS in this section.

Reply:
Also, if this section tells you to attach the ISO image, it should probably also tell you to optionally remove it.

Reply:
Updated